sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -5,6 +5,11 @@
 **BugFixes**
 
 * `forward_to` and `forward_dead_lettered_messages_to` will no longer cause authorization errors when used in `ServiceBusAdministrationClient` for queues and subscriptions.
+* Updated uAMQP dependency to 1.2.13.
+  - Fixed bug in accessing message properties triggering segmentation fault when the underlying C bytes are NULL.
+  - Fixed bug where connection establishment on macOS with Clang 12 triggering unrecognized selector exception.
+  - Fixed bug that macOS was unable to detect network error.
+  - Fixed bug that ReceiveClient and ReceiveClientAsync receive messages during connection establishment.
```

Comment:
Are there azure-sdk-python repo issue numbers we can reference here?

Reply:
updated, thanks for reminding.

btw, do you think it's a good idea to  mention those SB issues in the EH changelog as both SDKs depends on the same uamqp?